qemu/tests/x86_cpu_model.py

```diff
@@ -48,6 +49,8 @@ def run(test, params, env):
         flag_ib = " ibpb"
         name_ib = " \\(with IBPB\\)"
 
+    if add_suffix == 'no':
+        model_ib = 'unavailable'
```

Comment:
Would you pls explain this? What's this for?

Reply:
> Would you pls explain this? What's this for?

To avoid the situation 'Skylake-Server-v2-IBRS', there is no such model.

Reply:
It should have been avoided in L55~L61.

Reply:
> It should have been avoided in L55~L61.

It didn't avoid this because I can the get the invalid model, seems you judge this by some flags. and Skylake-Server-IBRS   is alias of Skylake-Server-v2, so we will get the invalid model.

Reply:
Did you have a try? There is no 'Skylake-Server-v2-IBRS' in query-cpu-definitions output. So `models` shouldn't contain the invalid model. And I don't understand why we get the invalid model due to alias issue. Would you pls explain more?

Reply:
> Did you have a try? There is no 'Skylake-Server-v2-IBRS' in query-cpu-definitions output. So `models` shouldn't contain the invalid model. And I don't understand why we get the invalid model due to alias issue. Would you pls explain more?

Yes, you are right, the error is caused by another reason, will update it, thanks.